src/document/DocumentAnnotator.scss

```diff
@@ -7,4 +9,10 @@
         width: 100%;
         pointer-events: none;
     }
+
+    &.is-highlighting {
+        .ba-TextLayer > span {
```

Comment:
We already refer to `.textLayer` in DocumentAnnotator, so it's probably fine to do so here, as well. We just really want to avoid any viewer-specific code in the React components and managers, since they're meant to be reusable.